public/testem/testem_client.js

```diff
@@ -127,6 +127,7 @@ var Testem = {
   emitMessageQueue: [],
   afterTestsQueue: [],
   console: {},
+  onceCallbackEventIdx: [],
```

Comment:
This name seems to imply _singular_, but is actually an array of indexes, no? Perhaps we should change to `onceCallbackEventIndexes`?